js/apps/admin-ui/src/groups/GroupTable.tsx

```diff
@@ -56,11 +56,11 @@ export const GroupTable = ({
     let groupsData = undefined;
     if (id) {
       groupsData = await fetchAdminUI<GroupRepresentation[]>(
-        "ui-ext/groups/subgroup",
-        { ...params, id },
+        `groups/${id}/children`,
```

Comment:
Same as with the other API calls, if this is now part of the official API it should be in the Admin Client instead.